lib/modules/onc_program/test/bulk_group_validation_sequence_test.rb

```diff
@@ -745,6 +745,12 @@
       p = @sequence.guess_profile(resource, :r4)
       assert p.url == Inferno::ValidationUtil::RESOURCES[:r4]['Device'].first.url
     end
+
+    it 'returns nil for Location' do
```

Comment:
what a weird unit test.

Reply:
Because Location uses FHIR base profile. Must assure the guess_profile does not pick  the US Core Location profile